templates/studentEnrollment.html

```diff
@@ -1,45 +1,226 @@
+							{y: $focus_count_pair[1], label: "$focus_area"},
+						{y: $focus_count_table[-1][1],  label: "$focus_count_table[-1][0]"}
+						]
+				  }]
+		});
+
+		focusAreaBarChart.render();
+
+		focusAreaPieChart = new CanvasJS.Chart("enrollment-focus-area-pie-chart",
+	    {
+			animationEnabled: true,
+			legend: {
+				verticalAlign: "center",
+				horizontalAlign: "left"
+			},
+			data: [{
+						type: "pie",
+						indexLabelFontFamily: "Garamond",       
+						indexLabelFontSize: 20,
+						indexLabelFontWeight: "bold",
+						startAngle:0,
+						indexLabelFontColor: "#161616",       
+						indexLabelLineColor: "#161616", 
+						indexLabelPlacement: "outside",
+
+						toolTipContent: "{name}: {y} (#percent%)",
+						showInLegend: false,
+						indexLabel: "{name}:  {y} (#percent%)", 
+						dataPoints: [      
+						$for focus_count_pair in focus_count_table[0:len(focus_count_table)-1]:
+							$ focus_area = focus_count_pair[0].replace('&', 'and')
+							{y: $focus_count_pair[1], name: "$focus_area", legendMarkerType: "square"},
+						{y: $focus_count_table[-1][1],  name: "$focus_count_table[-1][0]", legendMarkerType: "square"}
+						]
+				  }]
+		});
+	}
+
+	function showStudentYearBarChart() {
+		document.getElementById('enrollment-student-year-bar-chart').style.display = "block";
+		document.getElementById('enrollment-student-year-pie-chart').style.display = "none";
+		document.getElementById('enrollment-student-year-bar-chart-button').classList.add("active");
+		document.getElementById('enrollment-student-year-pie-chart-button').classList.remove("active");
+		studentYearBarChart.render();
+	}
+
+	function showStudentYearPieChart() {
+		document.getElementById('enrollment-student-year-pie-chart').style.display = "block";
+		document.getElementById('enrollment-student-year-bar-chart').style.display = "none";
+		document.getElementById('enrollment-student-year-pie-chart-button').classList.add("active");
+		document.getElementById('enrollment-student-year-bar-chart-button').classList.remove("active");
+		studentYearPieChart.render();
+	}
+
+	function showFocusAreaBarChart() {
+		document.getElementById('enrollment-focus-area-bar-chart').style.display = "block";
+		document.getElementById('enrollment-focus-area-pie-chart').style.display = "none";
+		document.getElementById('enrollment-focus-area-bar-chart-button').classList.add("active");
+		document.getElementById('enrollment-focus-area-pie-chart-button').classList.remove("active");
+		focusAreaBarChart.render();
+	}
+
+	function showFocusAreaPieChart() {
+		document.getElementById('enrollment-focus-area-pie-chart').style.display = "block";
+		document.getElementById('enrollment-focus-area-bar-chart').style.display = "none";
+		document.getElementById('enrollment-focus-area-pie-chart-button').classList.add("active");
+		document.getElementById('enrollment-focus-area-bar-chart-button').classList.remove("active");
+		focusAreaPieChart.render();
+	}
+</script>
```

Comment:
Similarly, is it possible to abstract all these out into another JavaScript file instead of writing them here?